.gitignore

```diff
@@ -72,3 +72,6 @@ Temporary Items
 
 ### Bower ###
 bower_components
+
+# .env with API keys
+.env
```

Comment:
Remove; we're storing in `local.env.json` like before. Updated version in Dropbox.